CONTRIBUTING.md

```diff
@@ -27,6 +27,8 @@ Check out the [security policy](https://github.com/NixOS/nix/security/policy).
 1. Search for related issues that cover what you're going to work on.
    It could help to mention there that you will work on the issue.
 
+   We strongly recommend first-time contributors not to propose new features but rather fix tightly-scoped problems in order to build trust and a working relationship with maintainers.
```

Comment:
Do you actually mean bugs when you tightly-scoped problems?
Because new features can also address tightly-scoped problems.

Reply:
The sentiment I got from the exchange that produced this statement is "don't grow the API surface". "Feature" may be misleading here, because what does that even mean exactly?